scargo/commands/test.py

```diff
@@ -44,6 +45,9 @@ def scargo_test(
     test_build_dir.mkdir(parents=True, exist_ok=True)
     conan_add_default_profile_if_missing()
 
+    conan_add_remote(project_dir, config)
```

Comment:
Bozidar Kresaj requested this change. Can you please clarify it with him?

Reply:
Okay, maybe I don't understand the issue actually. Let me clarify and I will get back to you

Reply:
OK, so what we need actually is to add conan_add_remote to test.py. 

The problem is that adding it to update.py will not do anything unless we commit the container. We can think about how to handle this in the future.
For this PR, we should not move the conan_add_remote to update as it will not do anything. Instead, add the call to the test command so the dependencies can be fetched from additional remotes.